BankSDK/GiniBankSDK/Pod/GiniBankSDK.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |spec|
+  spec.name               = "GiniBankSDK"
+  spec.version            = "3.7.1"
```

Comment:
```suggestion
  spec.version            = "3.7.2"
```

Reply:
I will do this manually now. But do we want to have this always up to date? It is supposed to be a model for the respective file from the `gini-podspecs` repo.

Reply:
We can skip to update it manually, at least we will know when we copied this podspec, for which SDK version 😅